.coveragerc

```diff
@@ -1,2 +1,4 @@
 [run]
-omit = sagemaker/tests/*, sagemaker/tensorflow/tensorflow_serving/*
+concurrency = threading
+omit = sagemaker/tests/*
+timid = True
```

Comment:
In case anyone else was curious, [docs](https://coverage.readthedocs.io/en/coverage-4.0.3/api_coverage.html):
> If `timid` is true, then a slower and simpler trace function will be used. This is important for some environments where manipulation of tracing functions breaks the faster trace function.